docusaurus/docs/protocol/tokenomics/token_logic_modules.md

```diff
@@ -0,0 +1,446 @@
+    TM2["Tokenomics Module"]
+
+    subgraph TLMGM["TLM: Global Mint"]
+        IA(["Inflation Mint Coin (IA)"])
+        ID["Inflation Distribution <br> (see TLM above for details)"]
+        GP["Governance Params"]
+
+        subgraph TO[Tokenomics Operations]
+            TM[[Tokenomics Module]]
+            IA("IA = SA * PCGI <br> (IA: Inflation Amount)")
+            TM  --> |"💲 MINT IA"|IA
+        end
+
+        %% Mint Inputs
+        SA --> TO
+        PCI --> TO
+        IA --> TO
+
+        %% Distribute Inflation
+        TO --> |"Distribute Inflation (IA) <br> ⬆️ INCREASE Balances"| ID
+        GP --> ID
+    end
+
+    subgraph AO[Application Operations]
+        AM[[Application Module]]
+        AK[(Application Keeper)]
+        AA[Application Address]
+
+        %% Reimbursement Request Actions
+        AM -.- AK
+        AM -. ⬇️ REDUCE Stake by IA .-> AA
+    end
+
+
+    %% Reimbursement Request Logic
+    TO ---> |Prevent Self Dealing <br> 🤝 HOLD IA| AO
+    AO -.-> |Emit Event| ARRE
+    AO --> |"⬆️ INCREASE Module Balance (IA)"| TM2
+
+    classDef module fill:#f9f,color: #333,stroke:#333,stroke-width:2px;
+    classDef address fill:#bbf,color: #333,stroke:#333,stroke-width:2px;
+    classDef govparam fill:#eba69a,color: #333,stroke:#333,stroke-width:2px;
+    classDef event fill:#e8b761,color: #333,stroke:#333,stroke-width:2px;
+
+    class TM,AM,TM2 module;
+    class PCI,GP govparam;
+    class PRA,AA,DAO address;
+    class ARRE event;
+```
+
+Later, PNF, on behalf of the DAO, will review the reimbursement requests and approve them.
+
+```mermaid
+---
+title: "Off-Chain Reimbursement Request Flow"
+---
+sequenceDiagram
+    participant PNF as Pocket Network Foundation
+    participant BS as Blockchain State
+    participant T as Tokenomics Module
+    participant Apps as Applications 1..N
+
+    PNF ->> +BS: Get All Reimbursement Requests
+    BS ->> -PNF: List of Reimbursement Requests
+    PNF ->> PNF: Review Reimbursement Requests
+    loop "for each request"
+        alt "Approve"
+            PNF ->> T: Reimburse Application Funds
+            T ->> Apps: Send Reimbursement
+        else "Reject"
+            note over PNF, Apps: PNF maintains funds
+        end
+    end
+```
+
+### FAQ
+
+#### Is the application going to endorse the whole global mint amount (Including the supplier share)? <!-- omit in toc -->
+
+The application `PAYS` the supplier for work done (i.e. Mint=Burn)
+The application `GETS REIMBURSED` for the inflation (i.e. Global Mint)
+
+#### Will there be any on-chain enforcement on the way application get reimbursed? Or do they just have to trust the DAO to reimburse them? <!-- omit in toc -->
+
+No. They have to trust the DAO/PNF reimburses them.
+
+This is similar to how we trust Gateways in Morse not to do self dealing.
+
+#### How does this solution scale for sovereign applications? Do they have to show face too? <!-- omit in toc -->
+
+Correct.
```

Comment:
I feel like this section needs to:
1. Rephrase or simplify the "question" 
AND/OR
1. Elaborate more in the "answer"

It seems like we're trying to make it clear that this applies to both gateways and sovereign applications. Perhaps adding a bit of detail that makes contact with why someone might believe otherwise would put it into better context.